src/room.ts

```diff
@@ -130,6 +130,9 @@ export class Room extends EventEmitter implements Sayable {
       const data = await contactGetter(this.id)
       log.silly('Room', `contactGetter(${this.id}) resolved`)
       this.rawObj = data
+      if (!this.rawObj.MemberList) {
+        throw new Error('no this.obj.memberList, set after contactGetter')
```

Comment:
As I said before: should NOT throw when `!this.rawObj.MemberList`.

Because this is OK.